src/encoded/static/components/item-pages/FileView.js

```diff
@@ -92,59 +92,53 @@ export default class FileView extends WorkflowRunTracingView {
 }
 
 
-class FileViewOverview extends React.PureComponent {
-
-    static getTabObject({ context, schemas, windowWidth, href }, width){
-        return {
-            'tab' : <span><i className="icon icon-file-alt fas icon-fw"/> Overview</span>,
-            'key' : 'file-overview',
-            //'disabled' : !Array.isArray(context.experiments),
-            'content' : (
-                <div className="overflow-hidden">
-                    <h3 className="tab-section-title">
-                        <span>More Information</span>
-                    </h3>
-                    <hr className="tab-section-title-horiz-divider"/>
-                    <FileViewOverview {...{ context, width, windowWidth, schemas, href }} />
-                </div>
-            )
-        };
-    }
 
-    static propTypes = {
-        'context' : PropTypes.shape({
-            'experiments' : PropTypes.arrayOf(PropTypes.shape({
-                'experiment_sets' : PropTypes.arrayOf(PropTypes.shape({
-                    '@id' : PropTypes.string.isRequired
-                }))
-            })),
+function FileViewOverview (props) {
+    const { context, windowWidth, width, schemas, href } = props;
```

Comment:
Codacy found an issue: ['windowWidth' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5937521)